test/Enable.Extensions.Queuing.RabbitMQ.Tests/RabbitMQTestFixture.cs

```diff
@@ -72,11 +74,14 @@ protected virtual void Dispose(bool disposing)
 
         private void DeleteQueue()
         {
-            using (var connection = _connectionFactory.CreateConnection())
-            using (var channel = connection.CreateModel())
+            var connection = _connectionFactory.CreateConnection();
```

Comment:
As above, why have the `using` statements been removed here? We should retain these so that `Dispose` is always called. With the current change, we're not disposing of the channel, nor would we dispose of the connection if `channel.QueuePurge` were to throw.

Reply:
The connection must also be closed before it is disposed https://www.rabbitmq.com/dotnet-api-guide.html#disconnecting. This has now been updated to use a try finally block to ensure the connection is closed and disposed.